packages/cli/package.json

```diff
@@ -21,7 +21,7 @@
     "node": ">=8.0.0"
   },
   "scripts": {
-    "setup:environment": "./import_contract_types_from_sdk.sh",
+    "setup:environment": "yarn --cwd=packages/contractkit build-sdk",
```

Comment:
Maybe remove this. We should assume sdk is built already. Or setup sdk directly, not from here

Reply:
I think it is good to setup SDK explicitly. Since in its current form, it might be set up for a different environment. `mobile` package has the same approach. I would recommend renaming `setup:environment` to `build-sdk` for consistency though.